src/Adapter/MSTest.TestAdapter/MSTest.TestAdapter.csproj

```diff
@@ -46,6 +47,11 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
 
+  <!-- Properties specific to UWP -->
+  <PropertyGroup Condition=" '$(TargetFramework)' == '$(UwpMinimum)' OR '$(TargetFramework)' == '$(ModernUwpMinimum)' ">
+  	<DefineConstants>$(DefineConstants);WINDOWS_UWP</DefineConstants>
+  </PropertyGroup>
```

Comment:
It seems unused at the moment, so we can drop it.

Reply:
Removed